store/src/main/java/com/dropbox/android/external/store4/impl/RealStore.kt

```diff
@@ -79,11 +79,16 @@ internal class RealStore<Key : Any, Input : Any, Output : Any>(
     )
 
     override fun stream(request: StoreRequest<Key>): Flow<StoreResponse<Output>> {
+        val cached = memCache?.get(request.key)
```

Comment:
why are we reading the cache here instead of onStart ?

Reply:
fixed